src/control/cmd/daos/snapshot.go

```diff
@@ -233,13 +233,14 @@ func (cmd *containerSnapListCmd) Execute(args []string) error {
 		return cmd.OutputJSON(snaps, nil)
 	}
 
-	cmd.Info("Container's snapshots :")
+	cmd.Infof("%s\t\t\t%s\t\t\t%s", "Timestamp" , "Epoch", "Name")
+	cmd.Infof("---------\t\t\t-----\t\t\t----")
```

Comment:
For consistency with other commands, it would be better to use the `txtfmt` package to format these. See [printAttributes()](https://github.com/daos-stack/daos/blob/master/src/control/cmd/daos/attribute.go#L49) for an example.

Reply:
fixed.